pages/api/v1/fantasy/index.ts

```diff
@@ -0,0 +1,87 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import SQL from 'sql-template-strings';
+import Cors from 'cors';
+import { query } from '../../../../lib/db';
+import use from '../../../../lib/middleware';
+
+const cors = Cors({
+  methods: ['GET', 'HEAD'],
+});
+
+export interface FantasyData {
+  seasonID: number;
+  position: string;
+  name: string;
+  fantasyPoints: number;
+}
+
+export default async (
+  req: NextApiRequest,
+  res: NextApiResponse<Array<FantasyData> | string>
+): Promise<void> => {
+  await use(req, res, cors);
+
+  const { season = null, position = null, name = null} = req.query;
+
+  const search = SQL`SELECT * FROM fantasypoints`
```

Comment:
logic to generate the filtering

Reply:
We should be able to get the most recent season from this table as the default when there is no `season` specified. Can we add the code from another api route to do this? It would clean up the filtering a ton. If the season is already defined we can just have it so `WHERE` is already in the query and seasonID is one of those conditions.